migrations/sql/V2017.11.13.1828__AddBurdenEstimateSetStatus.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE burden_estimate_set_status (
+  code TEXT NOT NULL ,
+  description TEXT NOT NULL ,
+  PRIMARY KEY (code)
+);
+COMMENT ON COLUMN burden_estimate_set_status.code IS 'Valid values: {empty, partial, complete}';
+
+INSERT INTO burden_estimate_set_status VALUES ('empty', 'No burden estimates added');
+INSERT INTO burden_estimate_set_status VALUES ('partial', 'Some but not all burden estimates added');
+INSERT INTO burden_estimate_set_status VALUES ('complete', 'Closed to further estimates');
+
+-- currently sets are populated at the same time as being completed, so set default status to complete
+ALTER TABLE burden_estimate_set
+  ADD COLUMN status TEXT DEFAULT 'complete' NULL,
```

Comment:
Shouldn't this be NOT NULL? All burden estimate sets should have a status. 

If you do change it to be non-nullable, make sure this works when applied to a database with existing data.

Reply:
oh yeah, good catch. should be fine cos of the default...